src/index.ts

```
@@ -7,8 +7,9 @@
 import * as commands from './json/models_by_command_id.json'
 import {AbangModel} from './types/model'
 import {createUser} from './api'
-import {generateCompletionWithCoins} from './services'
+import {generateCompletionSafely} from './services'
 import {addSpaceUser, getSpaceUser} from './db/team'
+import {callMessageApi} from './utils/googleapi'
```

ESLint

[error] 'callMessageApi' is defined but never used.

(no-unused-vars)